src/imgen.c

```diff
@@ -13,6 +13,11 @@
 #define STB_IMAGE_WRITE_IMPLEMENTATION
 #include <stb/stb_image_write.h>
```

Comment:
Can we add install instructions for stb_image to the READme?

Reply:
I used a package manager to install it -- but adding it to the default c lib path after downloading from source is effectively the full instructions

Reply:
Great!